qovery/resource_job_test.go

```diff
@@ -197,6 +201,9 @@ resource "qovery_job" "test" {
   {{ end }}
 
   healthchecks = {}
+
+  advanced_settings_json = jsonencode({"deployment.termination_grace_period_seconds":61})
```

Comment:
Doesn't work with:
```suggestion
  advanced_settings_json = jsonencode({{ .Job.AdvancedSettingsJson }})
```

Reply:
What does the template rendered looks like when using `advanced_settings_json = jsonencode({{ .Job.AdvancedSettingsJson }})`?

Reply:
It renders `advanced_settings_json = jsonencode("{\"deployment.termination_grace_period_seconds\":61}")` 
Instead of `advanced_settings_json = jsonencode({"deployment.termination_grace_period_seconds":61})`

Reply:
Ok I understand now, it's the inner value of the `types.String` that must be used ><